mautic/mautic
# Campaign tests

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | N
| Related user documentation PR URL | /
| Related developer documentation PR URL | /
| Issues addressed (#s or URLs) | /
| BC breaks? | N
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR creates first set of unit tests for the CampaignBundle. During writing the tests I found and fixed a few minor bugs and made some DRY improvements.

#### Steps to test this PR:
1. Apply this PR
2. Run `composer test`. All tests should pass.
3. Execute a campaign, everything should work as before.
